builders/build/eth-api/dev-env/openzeppelin/contracts.md

```diff
@@ -177,7 +177,7 @@ contract MyToken is ERC721, ERC721Enumerable, ERC721Burnable, Ownable {
 
```

Comment:
Line 166 Contract Wizard link should be updated to builders/build/eth-api/dev-env/openzeppelin/contracts/#openzeppelin-contract-wizard